core/templates/dev/head/profile/Profile.js

```diff
@@ -14,6 +14,8 @@
 
 /**
  * @fileoverview Data and controllers for the Oppia profile page.
+ *
+ * @author sean@seanlip.org (Sean Lip)
```

Comment:
Please remove these two lines -- we've removed author lines throughout the codebase, since files tend to be edited by multiple people.